src/Connector/APIWrapper.php

```diff
@@ -573,35 +573,36 @@ public function authenticateCustomer($email, $password) {
   /**
    * {@inheritdoc}
    */
-  public function getCustomer($email, $throwRouteException = TRUE) {
+  public function getCustomer($email, $throwCustomerNotFound = TRUE) {
```

Comment:
This is breaking BC compatibility. Besides, I don't see any changes in calls to this function.

Reply:
I propose breaking backwards compatibility is worthwhile. We only have two customers and both are managed by Acquia PS. It is intended to add more custom exceptions in the future. There is no change in the calling function because the calling function catches \Exception and not \CustomerNotFoundException.